core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.ts

```diff
@@ -135,6 +140,35 @@ angular.module('oppia').controller('QuestionSuggestionReviewModalController', [
         });
     };
 
+    $scope.edit = function() {
+      SkillBackendApiService.fetchSkillAsync(
+        suggestion.change.skill_id).then((skillDict) => {
+        var skill = skillDict.skill;
+        $uibModal.open({
```

Comment:
Do we need to open a modal here? Can't we just set `$scope.canEditQuestion` to `true` so that the user can edit the suggestion inline and surface an "update" and "cancel" button like we do with the translation suggestions? We should make the experience close to the translation suggestion editing experience.